doc/15-troubleshooting.md

```diff
@@ -176,6 +176,60 @@ C:\> cd C:\ProgramData\icinga2\var\log\icinga2
 C:\ProgramData\icinga2\var\log\icinga2> Get-Content .\debug.log -tail 10 -wait
 ```
 
+### Enable/Disable Debug Output on the fly <a id="troubleshooting-enable-disable-debug-output-api"></a>
+
+Every feature is just an Icinga config object similar to Host and Service, e.g.
+the `debuglog` feature is a [FileLogger](09-object-types.md#objecttype-filelogger).
+Those can also be [managed via API](12-icinga2-api.md#icinga2-api-config-objects)
+at runtime. This is a good alternative to `icinga2 feature enable debuglog`:
+
+* Object creation/deletion via API happens immediately and requires no restart
+* Hence, the debug log is enabled exactly as long as desired
+
+However, in case of [a HA zone](06-distributed-monitoring.md#distributed-monitoring-scenarios-ha-master-agents)
+the feature gets enabled/disabled on both nodes once they're connected.
```

Comment:
Since this is like any other configuration object, is it worth mentioning some HA specific details? If in doubt, I would rephrase it slightly different, as I can currently think of it as both masters need to be waiting to each other to be connected in order to get enabled the `debuglog` on both sides.

Reply:
> Since this is like any other configuration object, is it worth mentioning some HA specific details?

Given that feature's I/O impact, no. It's not as trivial as every other object.

> I can currently think of it as both masters need to be waiting to each other to be connected in order to get enabled the `debuglog` on both sides.

Exactly! As long as the other master isn't connected, it doesn't write the debuglog.

Reply:
> Exactly! As long as the other master isn't connected, it doesn't write the debuglog.

Seriously? How? Why?

Reply:
I'm sorry.

~As long as the other master isn't connected, it doesn't write the debuglog.~

As long as the other master isn't connected, it doesn't start writing the debuglog too as it didn't get the runtime debug logger yet.

Reply:
> As long as the other master isn't connected, it doesn't start writing the debuglog too as it didn't get the runtime debug logger yet.

I'm not questioning that! I'm just confused, and maybe others will be too, by this one `... the feature gets enabled/disabled on both nodes once they're connected.` The one specific endpoint that receives the API request does not have to wait for anyone to enable/disable it.